doc/user/methods/random_forest.md

```diff
@@ -705,10 +716,107 @@ class CustomCategoricalSplit
 
 #### `UseBootstrap`
 
+***Note:*** this parameter will be removed in mlpack 5.0.0. A value of `false`
+will then be equivalent to setting `BootstrapType` to `IdentityBootstrap`, and a
+value of `true` will be equivalent to `DefaultBootstrap`.
+
  * A `bool` value that indicates whether or not a bootstrap sample of the
    dataset should be used for the training of each individual decision tree in
    the random forest.
  * If `true` _(default)_, a different bootstrap sample of the same size as the
    dataset will be used to train each decision tree.
  * If `false` _(default for the `ExtraTrees` [variant](#fully-custom-behavior))_, the full
    dataset will be used to train each decision tree.
+
+#### `BootstrapType`
+
+ * Specifies the strategy used for bootstrapping data for each tree in the random forest.
+ * Three implementations for `BootstrapType` are available for drop-in usage:
+   - `DefaultBootstrap` *(default)*: bootstrap via random sampling with replacement.
+   - `IdentityBootstrap`: no bootstrapping.  Simply copies the input `dataset`, `labels`, and `weights` for each tree's data.
+   - `SequentialBootstrap`: bootstrapping from overlapping sequences such that samples with informational overlap behave more I.I.D.
+     * Useful when data consists of multiple overlapping events (or individual sequences).
+     * `b = SequentialBootstrap(intervals)` will create a `SequentialBootstrap` object, where:
+       - `intervals` is of type `arma::umat`, with 2 rows and `n` columns, where `n` is the number of events to be sampled from.
+       - Each column in `intervals` represents the start and end columns (inclusive) of each event.
+       - So, e.g., if the 10th event is 5 points long, starting at index 6, then column `9` of `intervals` should be `[6, 10]`.
+     * A `SequentialBootstrap` must be passed as the `bootstrap` option to the [advanced constructor](#fully-custom-behavior).
```

Comment:
```suggestion
     * A `SequentialBootstrap` must be passed as the `bootstrap` option to the advanced constructor (below).
```
That href doesn't actually point anywhere useful---since we introduce the constructor below, better to just say "below".  Sorry if that was originally a suggestion of mine that wasn't good.